build.gradle

```diff
@@ -76,7 +77,7 @@ subprojects {
 
   apply plugin: 'java'
   apply plugin: 'checkstyle'
-  apply plugin: 'findbugs'
+  apply plugin: "com.github.spotbugs"
```

Comment:
Should we extract the migration from findbugs to spotbugs on a separate PR? This would let us avoid `finished with non-zero exit value 137` error from blocking us.

Reply:
circleci failures seems to be due to OutOfMemory exceptions when running build. Most probably due to increased spotbugs memory needs.

Are you suggesting migrating first to spotbugs and keep java8 compiler? I think we'd hit the same issue

Reply:
Splitting spotbugs in separate runs seems to workaround the OOM issue in circleci.
@efeg I am thinking of creating a "verify" gradle task to aggregate these two tasks. Would this be a decent workaround?